src/hospital/PatientCollection.java

```diff
@@ -0,0 +1,46 @@
+package hospital;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ *
+ * @author maximdumont
+ */
+public class PatientCollection{
+    private List<Patient> patients = new ArrayList<>();
+    
+    public PatientCollection(){
+        
+    }
+    
+    public PatientCollection(List<Patient> patients){
```

Comment:
I'd prefer to construct this without passing in an externally-generated ArrayList, since they're passed by reference and vulnerable to being modified externally. You have an add method for List interface collections so we can use that when needed.